libraries/libfc/test/crypto/test_webauthn.cpp

```diff
@@ -207,14 +206,11 @@ BOOST_AUTO_TEST_CASE(challenge_base64_dot_padding) try {
    });
 } FC_LOG_AND_RETHROW();
 
-//valid signature but remove padding
+//valid signature
```

Comment:
This test is now just repeating what the `good` test does, but more importantly I believe it means we no longer have a test with valid `=` padding. I think instead this test should be modified to append valid padding to ensure that base64urldecode ignores padding as required.